src/Units/UnitModel.java

```diff
@@ -15,11 +15,10 @@
     INFANTRY, MECH, RECON, TANK, MD_TANK, NEOTANK, APC, ARTILLERY, ROCKETS, ANTI_AIR, MOBILESAM, FIGHTER, BOMBER, B_COPTER, T_COPTER, BATTLESHIP, CRUISER, LANDER, SUB
   };
 
-  // NB: "Truck" is technically unnecessary, but I feel it may be useful for alternate damage systems.
-  // The rubric for what's a "truck" is whether the damage dealt to it by an infantry is >9%, not movetype.
+  // Subs are ships unless they're submerged.
   public enum ChassisEnum
   {
-    TROOP, TRUCK, TANK, SHIP, AIR_LOW, AIR_HIGH
+    TROOP, TANK, AIR_LOW, AIR_HIGH, SHIP, SUBMERGED
```

Comment:
Why no more TRUCK? Just not enough to differentiate it from tanks?

Reply:
Basically, yes.